index.html

```diff
@@ -84,10 +262,325 @@ <h4>Monday - Friday, July 19 - July 23, 2021</h4>
         <div class="invalid-feedback">Please provide your affiliation.</div>
       </div>
 
+      <hr>
+      <h5>LSST/DESC Membership</h5>
```

Comment:
```suggestion
      <h5>LSST DESC Membership</h5>
```
But why are we asking for this again?